### ISSUE TITLE
Vector types issue

### ISSUE THREAD

**Issue by FlashHit** [Imposter/vscode-lua-vu#12]
Instead of just being a vector it should be also the type. ie: 
`SoundPatchAsset.outputNodes` will only show it as a `vector` instead of `vector|AudioGraphNodeData[]`

**Comment by FlashHit** [comment]
so instead of 
```
---@class SoundPatchAsset:SoundGraphAsset
---@field outputNodes vector @ **Vector Type: `AudioGraphNodeData`**
```

it should be 

```
---@class SoundPatchAsset:SoundGraphAsset
---@field outputNodes vector|AudioGraphNodeData[]
```

**Comment by Imposter** [comment]
Fixed in 44dd71d
